src/ocean/util/app/ext/LogExt.d

```diff
@@ -249,3 +248,17 @@ class LogExt : IConfigExtExtension
         return LogUtil.newLayout(name);
     }
 }
+
+
+/// Hack to allow to call a deprecated function without deprecations
+private extern extern(C) void configureOldLoggersLogExtHack (
+    ClassIterator!(LogUtil.Config, ConfigParser) config,
+    LogUtil.MetaConfig m_config,
+    AppenderExternD file_appender, MakeLayoutExternD makeLayout,
+    bool use_insert_appender = false);
+/// Hack because DMD1 applies extern(C) to the delegate type as well...
+private alias extern(D) Appender delegate (istring, Appender.Layout)
+    AppenderExternD;
+/// Ditto
+private alias extern(D) Appender.Layout delegate (cstring)
+    MakeLayoutExternD;
```

Comment:
Only one request - can you add a comment here with sub-string "deprecated" so that it pops up on regular cleanup when releasing 4.0.0 ?